xrpl/wallet/main.py

```diff
@@ -16,69 +16,141 @@ class Wallet:
+        """
+        The address that publicly identifies this wallet,
+        as a base58 string.
+        """  # noqa: DAR201
+        return self._classic_address
+
+    address = classic_address
+    """
+    The address that publicly identifies this wallet, as a base58 string.
+    """
+
     def __init__(
         self: Wallet,
-        seed: str,
-        sequence: int,
+        public_key: str,
+        private_key: str,
         *,
-        algorithm: Optional[CryptoAlgorithm] = None,
+        master_address: Optional[str] = None,
+        seed: Optional[str] = None,
     ) -> None:
         """
         Generate a new Wallet.
 
         Args:
-            seed: The seed from which the public and private keys are derived.
-            sequence: The next sequence number for the account.
-            algorithm: The algorithm used to encode the keys. Inferred from the seed if
-                not included.
+            public_key: The public key for the account.
+            private_key: The private key used for signing transactions for the account.
+            master_address: Include if a Wallet uses a Regular Key Pair. This sets the
+                address that this wallet corresponds to. The default is `None`.
+            seed: The seed used to derive the account keys. The default is `None`.
         """
-        self.seed = seed
-        """
-        The core value that is used to derive all other information about
-        this wallet. MUST be kept secret!
-        """
-
-        pk, sk = derive_keypair(self.seed, algorithm=algorithm)
-        self.public_key = pk
+        self.public_key = public_key
         """
         The public key that is used to identify this wallet's signatures, as
         a hexadecimal string.
         """
 
-        self.private_key = sk
+        self.private_key = private_key
         """
         The private key that is used to create signatures, as a hexadecimal
         string. MUST be kept secret!
         """
 
-        self.classic_address = derive_classic_address(self.public_key)
-        """The address that publicly identifies this wallet, as a base58 string."""
+        self._classic_address = (
+            ensure_classic_address(master_address)
+            if master_address is not None
+            else derive_classic_address(self.public_key)
+        )
+        """Internal variable for classic_address. Use classic_address instead."""
 
-        self.sequence = sequence
+        self.seed = seed
         """
-        The next available sequence number to use for transactions from this
-        wallet.
-        Must be updated by the user. Increments on the ledger with every successful
-        transaction submission, and stays the same with every failed transaction
-        submission.
+        The core value that is used to derive all other information about
+        this wallet. MUST be kept secret!
         """
 
     @classmethod
     def create(
```

Comment:
Should this be renamed to `generate` to mimic xrpl.js? 

I think it's fine to leave this as `create`, so that it's the same as previous.

Reply:
I mentioned it. Jackson said it was better to leave it as create for continuity.